mutexbot/reserve/action.yml

```diff
@@ -0,0 +1,59 @@
+name: "MutexBot Reserve"
+description: "Reserve a resource on MutexBot"
+inputs:
+  api-key:
+    description: "API Key"
+    required: true
+  resource-name:
+    description: "Name of the resource to reserve"
+    required: true
+  duration:
+    description: "Duration for the reservation"
+    required: false
+    default: 24h
+runs:
+  using: "composite"
+  steps:
+    - name: "Reserve resource"
```

Comment:
Let's add a timeout for this step? (I'm ok with input or a meaningful hardcoded value)

Reply:
We don't need to add a timeout to the action, because github already has a way to timeout any step